script/playercomponent.js

```diff
@@ -216,8 +216,8 @@ define(
         var hasNextCDN = mediaMetaData.urls.length > 1;
         var aboutToEndVod = getDuration() > 0 && (getDuration() - getCurrentTime()) <= 5;
         var canVodFailover = windowType === WindowTypes.STATIC && !aboutToEndVod;
-        var canHlsLiveFailover = windowType === WindowTypes.GROWING && (liveSupport === LiveSupport.SEEKABLE || liveSupport === LiveSupport.PLAYABLE) && transferFormat === 'hls';
-        var canDashLiveFailover = windowType !== WindowTypes.STATIC && transferFormat === 'dash';
+        var canHlsLiveFailover = windowType === WindowTypes.GROWING && (getLiveSupport() === LiveSupport.SEEKABLE || getLiveSupport() === LiveSupport.PLAYABLE) && transferFormat === TransferFormats.HLS;
```

Comment:
These `getLiveSupport` calls need device passed in in case we're using tal strategy

Reply:
👍 good spot